.github/workflows/release-package.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Package
+
+on:
+  release:
+    types: [created]
+  push:
+    branches:
+      - main
```

Comment:
@baldarn I think in this way every time we merge on main we publish a new version, I don't think it's correct.